src/System.Drawing.Primitives/src/System/Drawing/Rectangle.cs

```diff
@@ -0,0 +1,501 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics.Contracts;
+using System.Globalization;
+
+namespace System.Drawing
+{
+    /// <devdoc>
+    ///    <para>
+    ///       Stores the location and size of a rectangular region. For
+    ///       more advanced region functions use a <see cref='System.Drawing.Region'/>
```

Comment:
Remove the reference to System.Drawing.Region as we aren't porting that one forward.